src/Chronos.hs

```diff
@@ -248,7 +249,9 @@ import Data.Int (Int64)
 import Data.Char (isDigit)
 import Data.ByteString (ByteString)
 import Torsor (add,difference,scale,plus)
-import Chronos.Internal.CTimespec (getPosixNanoseconds)
```

Comment:
Why is this import necessary?

Reply:
That was a typo from before switching to Win32.